abi3audit/_audit.py

```diff
@@ -112,7 +119,8 @@ def audit(so: SharedObject, assume_minimum_abi3: PyVersion = PyVersion(3, 2)) ->
                 if maybe_abi3.added > baseline:
                     future_abi3_objects.add(maybe_abi3)
             elif sym.name.startswith("Py_") or sym.name.startswith("_Py_"):
-                non_abi3_symbols.add(sym)
+                if sym.name not in _ALLOWED_SYMBOL_NAMES:
+                    non_abi3_symbols.add(sym)
```

Comment:
In practice I think this is fine, but there's still a subtle edge case here 😅 -- we *do* want to reject `Py_XDECREF` if it's a global/public symbol, since that indicates that it's being linked from `libpython` and might not conform to the stable ABI.

In practice I'm pretty sure this is impossible anyways, since `Py_XDECREF` is `static inline` and it certainly wouldn't be the only abi3 violation if it was one. So I'm fine with this for now.

Reply:
I'm guessing we could control this by setting the symbol visibility, like mentioned in point 2) of the linked issue?

Reply:
Yep, exactly what I was thinking.